contrib/init/upstart/docker.conf

```diff
@@ -10,6 +10,10 @@ respawn
 kill timeout 20
 
 pre-start script
+	mountpoint() {
+		awk '{ print $2 }' /proc/mounts | grep -q ^${1}$
```

Comment:
can get away without the grep:

    ! awk -v mp="$1" '$2 == mp { exit 1 }' /proc/mounts